tests/testthat/test-roche_pvalue.R

```diff
@@ -5,3 +5,11 @@ test_that("roche_pvalue() works", {
     c(NA, "<0.0001", "<0.0001", "0.5000", "0.9000", "1.0000", NA)
   )
 })
+
+test_that("roche_percent() works", {
+  expect_equal(
+    c(1, 0.0008, 0.9998, 0.998, 0.5, -1, 1.1) |>
```

Comment:
Can you add tests varying the `digits` argument?

Reply:
Good call - I've added a few tests with different digits